device/accton/x86_64-accton_wedge100bf_32qs-r0/platform.json

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
To avoid platform TCs issues, I'd recommend to add chassis info as well. For more details, please refer to https://github.com/Azure/sonic-buildimage/blob/master/device/barefoot/x86_64-accton_wedge100bf_32x-r0/platform.json